atomicapp/cli/main.py

```diff
@@ -37,10 +37,19 @@
 
 logger = logging.getLogger(__name__)
 
+APP_MESSAGE = """
+Your application resides in %s
+Please use this directory for managing your application
+"""
+
 
 def cli_install(args):
     try:
-        NuleculeManager.do_install(**vars(args))
+        argdict = args.__dict__
+        nm = NuleculeManager(app_spec=argdict['app_spec'],
+                             destination=argdict['destination'])
+        nm.install(**argdict)
+        print(APP_MESSAGE % nm.app_path)  # msg for users
```

Comment:
I decided specifically to not use logging.info mainly because the output is so verbose that people tend to ignore it (at least I do). Either that or it becomes a chore to sift through it to find something useful. 

I made a decision to spit this out as the last thing that gets output and separate from the other lines so that people would read it and say "Oh, ok.. that's where my application files are". I think what we'll find is that it is much better for user experience.